### PR TITLE
Add key signatures

### PR DESCRIPTION
**Pull request by kaseyknudsen**
### **Summary**
This PR enhances the visibility of the Snackbar and incorporates key signatures into the "Identify Key Signatures" page.

**Key Changes**

- Updated Snackbar: Adjusted placement and color to improve visibility.
- Added Key Signatures: Introduced the four key signatures, each displayed with a double bar line.
- Code Cleanup: Removed unused code to streamline the codebase.
- Updated Buttons: Enabled buttons remain yellow to improve UI

**Impact**
The Snackbar is now more noticeable for users, the enabled buttons remain highlighted, and the key signatures are clearly presented, enhancing the overall user experience.

### CODE REVIEW

File: app/(routes)/exam/page.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 "use client";
 import ChordsIdentify from "@/app/components/ExamPages/ChordsIdentify";
 import KeySigIdentify from "@/app/components/ExamPages/KeySigIdentify";
+import KaseyKeySigIdentify from "@/app/components/ExamPages/KeySigIdentify";
```
Comment (BrettEastman) [inline app/(routes)/exam/page.tsx:4]: You could remove KaseyKeySigIdentify and just leave KeySigIdentify. Then change the name of the KaseyKeySigIdentify component.

File: app/(routes)/exam/page.tsx [hunk 2]
```diff
@@ -351,7 +352,7 @@ export default function ExamHomePage() {
           />
         )}
         {viewState === VIEW_STATES.KEY_SIG_IDENTIFY && (
-          <KeySigIdentify
+          <KaseyKeySigIdentify
```
Comment (BrettEastman) [inline app/(routes)/exam/page.tsx:355]: Change this one to KeySigIdentify to match import above.

File: app/components/CreateBlankStaves.tsx
Comment (BrettEastman) [inline app/components/CreateBlankStaves.tsx:1]: There is a CreateBlankStavesNew also, so it seems one of them needs to be removed?

File: app/components/SnackbarToast.tsx [hunk 3]
```diff
@@ -31,16 +34,42 @@ export default function SimpleSnackbar({
...
+        left: "50%",
+        width: "100%",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <SnackbarContent
+        style={{
+          backgroundColor: "red",
```
Comment (BrettEastman) [inline app/components/SnackbarToast.tsx:62]: I think the red is a bit intense. What do you think about the yellow that we use for the progress bar? --colorFilledProgressBarColor: #fde55b;

**Review by BrettEastman — APPROVED** [review]
Wrote a few comments to review, but after that it will be good to go!

File: app/(routes)/exam/page.tsx (hunk 1 above)
Comment (kaseyknudsen) [reply app/(routes)/exam/page.tsx:4]: oops, i meant to do that

File: app/components/SnackbarToast.tsx (hunk 3 above)
Comment (kaseyknudsen) [reply app/components/SnackbarToast.tsx:62]: i think an error code needs to be red, but we can make it a lighter red

File: app/components/CreateBlankStaves.tsx
Comment (kaseyknudsen) [reply app/components/CreateBlankStaves.tsx:1]: yes, but right now both are being used in several components. I'll refactor and create a separate PR for that eventually. I was getting a bunch of errors and it was a quick fix.

**Pull request merged**